protocol/bin/flow.js

```diff
@@ -3,38 +3,38 @@
 var promise = require('bluebird')
 var fs = promise.promisifyAll(require('fs'))
 var path = require('path')
-var root = 'json'
 
-fs
-.readdirAsync(root)
-.filter(jsonOnlySkipGregor)
-.map(load)
-.map(analyze)
-.reduce((acc, typeDefs) => acc.concat(typeDefs), [])
-.then(t => t.sort())
-.then(makeRpcUnionType)
-.then(write)
-
-let incomingMaps = {}
+var projects = [
```

Comment:
trying to keep things simple here. just pulling in this other file and working on it. this is unlikely to change anytime soon